Makefile

```
@@ -136,6 +151,12 @@ clean/keychain:
 clean/build:
 	rm -rf build/ release/ $$out
 
+.PHONY: clean/mutagen
+clean/mutagen:
+	rm -f $(PROJECT)/Resources/mutagen-agents.tar.gz
+	rm -f $(PROJECT)/Resources/mutagen-darwin-arm64
+	rm -f $(PROJECT)/Resources/mutagen-darwin-amd64
```

Feel free to ignore:

```suggestion
clean/mutagen:
	find "$(PROJECT)/Resources" -name 'mutagen-*' -delete
```